DemoApp/Sources/Views/JoinCallView.swift

```diff
@@ -14,30 +14,24 @@ struct JoinCallView: View {
     @State private var callId = ""
 
     var body: some View {
-        VStack {
-            Text("Join Call")
-                .font(.title)
-                .frame(maxWidth: .infinity, alignment: .leading)
-                .padding([.bottom])
-
-            VStack(spacing: 16) {
-                TextField("Enter call id", text: $callId)
-
-                Button {
-                    presentationMode.wrappedValue.dismiss()
-                    viewModel.joinCallAnonymously(callId: callId, completion: completion)
-                } label: {
-                    Text("Join call")
+        NavigationView {
```

Comment:
To get a navigationBar and not having a custom view to imitate it.

Reply:
Mostly because it's deprecated from iOS 16, but I don't think someone will change our deployment target and do that. All good, let's go with the navigation view.